### PR TITLE
make the struct Properties embedded into config Ref#933

### PR DESCRIPTION
**Pull request by tanjunchen**
### Ⅰ. Describe what this PR did
remove  the repeat struct Properties 
### Ⅱ. Does this pull request fix one issue?

Ref #933

### CODE REVIEW

**Comment by tanjunchen** [comment]
@yeya24  Ok

File: cmd/dfget/app/root.go [hunk 1]
```diff
@@ -104,36 +101,36 @@ func checkParameters() error {
 	return nil
 }
 
-// initProperties loads config from property files.
-func initProperties() {
-	properties := config.NewProperties()
+// initConfig loads config from property files.
+func initConfig() {
```
Comment (Starnop) [inline cmd/dfget/app/root.go]: I haven't found a place to call this function?

**Comment by Starnop** [comment]
IMOP, it's OK to make the struct Properties embedded  into `config`. WDYT? @yeya24 @tanjunchen

**Comment by yeya24** [comment]
@tanjunchen  Please change the PR name. Tell what you will do in this PR in the title instead of `fix #xxx`

**Comment by tanjunchen** [comment]
@yeya24 OK

**Comment by Starnop** [comment]
@tanjunchen Any updates?

**Comment by Starnop** [comment]
LGTM.

File: dfget/config/config.go [hunk 2]
```diff
@@ -240,6 +219,9 @@ type Config struct {
 
 	// The reason of backing to source.
 	BackSourceReason int `json:"-"`
+
+	// Properties
+	Properties
```
Comment (yeya24) [inline dfget/config/config.go]: Please add comments here.

Comment (tanjunchen) [reply dfget/config/config.go]: OK THX

File: dfget/config/constants.go [hunk 3]
```diff
@@ -53,6 +53,7 @@ const (
 	DefaultLocalLimit      = 20 * rate.MB
 	DefaultMinRate         = 64 * rate.KB
 	DefaultClientQueueSize = 6
+	DefaultTotalLimit      = 20 * rate.MB
```
Comment (yeya24) [inline dfget/config/constants.go]: Put this line above

Comment (tanjunchen) [reply dfget/config/constants.go]: OK

Comment (tanjunchen) [reply dfget/config/constants.go]: You mean Line  DefaultTotalLimit      = 20 * rate.MB  is placed in front of line  DefaultClientQueueSize = 6  
such as:
DefaultTotalLimit      = 20 * rate.MB
DefaultClientQueueSize = 6

Comment (yeya24) [reply dfget/config/constants.go]: Right below the line of locallimit

**Comment by Starnop** [comment]
LGTM.

**Pull request merged**